### PR TITLE
Implement BaseSecure on temporary admin restricted pages

### PR DESCRIPTION
**Pull request by jujaga**
# Description

* [SHOWCASE-1655](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1655) Enable deployment to PROD
* [SHOWCASE-1658](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1658) Implement BaseSecure on temporary admin restricted pages
* [SHOWCASE-1659](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1659) Hide /app/search
* [SHOWCASE-1660](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1660) Fix "annonymous" typo

## Types of changes

Bug fix (non-breaking change which fixes an issue)
New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-70

**Review by loneil — APPROVED** [review]
image

**Review by TimCsaky — APPROVED** [review]
Looks great

File: app/frontend/src/components/base/BaseSecure.vue [hunk 1]
```diff
@@ -1,21 +1,41 @@
...
+          </v-btn>
+        </router-link>
+      </div>
     </div>
+    <!-- TODO: Figure out better way to alert when user lacks chefs user role -->
     <div v-else class="text-center">
-      <h1 class="my-8">You are not authorized to use this feature.</h1>
-      <router-link :to="{ name: 'Home' }">
+      <h1 class="my-8">
+        Your account is not set up correctly. Please contact support.
```
Comment (parc-jason) [inline app/frontend/src/components/base/BaseSecure.vue:20]: Should ask Matt/Chris what to put here, it should have some contact info.  Not good to say contact support without any contact information.

File: app/frontend/src/views/Search.vue [hunk 2]
```diff
@@ -1,7 +1,9 @@
 <template>
-  <v-container>
-    <h1 class="my-6 text-center">Search Forms (TBD)</h1>
-  </v-container>
+  <BaseSecure>
```
Comment (parc-jason) [inline app/frontend/src/views/Search.vue:2]: why aren't we just completely removing this and the path?  seems silly to keep this as a hittable route, it should 404 not 401.  we don't have any significant source code to retain, no reason to keep this.

**Review by parc-jason — CHANGES_REQUESTED** [review]
Like the lockdown! We should clean up some dead code (it not now, then when?) and really should get appropriate support contact information for that page.

File: app/frontend/src/views/Search.vue (hunk 2 above)
Comment (jujaga) [reply app/frontend/src/views/Search.vue]: Dropped the search page. Easy enough to re-introduce if/when we decide to add that type of feature for rudimentary form discovery.

File: app/frontend/src/components/base/BaseSecure.vue (hunk 1 above)
Comment (jujaga) [reply app/frontend/src/components/base/BaseSecure.vue]: Updated as per discussions.

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-70

**Review by parc-jason — APPROVED** [review]
👍 👍

**Pull request merged**